### ISSUE TITLE
Support helm plugins

### ISSUE THREAD

**Issue by Sebor** [werf/werf#4394]
Is there any plan to add support of helm plugins?
In particular [helm-secrets](https://github.com/jkroepke/helm-secrets)?
At the present time I can invoke plugin commands but they don't work (nothing happens).

```bash
$ werf helm plugin list

NAME    VERSION DESCRIPTION                                                                  
diff    3.4.2   Preview helm upgrade changes as a diff                                       
secrets 3.13.0  This plugin provides secrets values encryption for Helm charts secure storing
```

```bash
$ werf helm secrets view secrets.yaml

$ echo $?
0
```

Werf version: 1.2.88

**Comment by Sebor** [comment]
### Additional info
with `WERF_HELM3_MODE=1` it does not work as expected:
- there is usage info at the end of output
  ```
  Usage:
    helm upgrade [RELEASE] [CHART] [flags]
  
  Flags:
        --add-annotation stringArray   Add annotation to deploying resources (can specify multiple).
                                       Format: annoName=annoValue.
                                       Also, can be specified with $WERF_ADD_ANNOTATION_* (e.g. $WERF_ADD_ANNOTATION_1=annoName1=annoValue1, $WERF_ADD_ANNOTATION_2=annoName2=annoValue2)
        --add-label stringArray        Add label to deploying resources (can specify multiple).
                                       Format: labelName=labelValue.
                                       Also, can be specified with $WERF_ADD_LABEL_* (e.g. $WERF_ADD_LABEL_1=labelName1=labelValue1, $WERF_ADD_LABEL_2=labelName2=labelValue2)
        --atomic                       if set, upgrade process rolls back changes made in case of failed upgrade. The --wait flag will be set automatically if --atomic is used
        --ca-file string               verify certificates of HTTPS-enabled servers using this CA bundle
       ....
  ```
- flag `--namespace` is ignored

**Comment by Sebor** [comment]
Update:

Using helm plugins (tested with `helm-secrets`) has been fixed in version [1.2.111](https://github.com/werf/werf/releases/tag/v1.2.111).

But the bug about ignoring `-n / --namespace` flag still presents.

**Comment by Sebor** [comment]
Fixed in versions 1.2.111 and 1.2.114

**Comment by vgolovnin** [comment]
After version 1.2.111 any non-zero exit code on helm plugin leads to whole process termination. But there are cases when non-zero exit code doesn't mean an error. For example helm diff with --detailed-exitcode param exits non-zero when changes exists (used in helmfile) @distorhead

**Comment by evgkrsk** [comment]
Yep, seems like helm-mode usage with helmfile broken with 1.2.144:

```
> helmfile -e monitoring -f monitoring.yaml deps
panic: error determining helm version: command "/opt/homebrew/bin/werf" exited with non-zero status:

PATH:
  /opt/homebrew/bin/werf

ARGS:
  0: werf (4 bytes)
  1: version (7 bytes)
  2: --client (8 bytes)
  3: --short (7 bytes)

ERROR:
  exit status 1

EXIT STATUS
  1

STDERR:
  Error: unknown flag: --client
```

**Comment by distorhead** [comment]
@vgolovnin @evgkrsk Hello! Issue with detailed-exitcode has been resolved by the https://github.com/werf/werf/pull/4537, fix available since v1.2.115 release (https://github.com/werf/werf/releases/tag/v1.2.115, v1.2 alpha or beta channels at the moment).
